site/profile/files/fragments/docs.html

```diff
@@ -0,0 +1,111 @@
+  <article class="content">
+    <h2>Help</h2>
+
+    <p>This test environment will ensure that your browser meets minimum requirements
+      and that your network configuration allows you to access all the services we run
+      in the classroom. If the test fails to launch when you click the button, then
+      please inform us of the issue by responding to the email you received or by
+      directly emailing <a href="mailto:eduteam@puppet.com">eduteam@puppet.com</a>.</p>
+
+    <h2>FAQ</h2>
+
+    <ul id="faq">
+      <li>
+        <h4>Where can I find instructions for using the classroom tools?</h4>
+        <div class="answer">
+          <p>We use a tool that allows each student to perform classroom exercises by writing code
+            in a graphical editor and enforcing it on a virtualized agent node. Read the
+            <a href="classroom_tutorial.html">tutorial</a>.</p>
+        </div>
+      </li>
+      <li><h4>Where can I find tutorials on using the Git server?</h4>
+        <div class="answer">
+          <p>We run a self-hosted Git server called <a href="https://gitea.io">Gitea</a>.
+            While it does not have very good end-user documentation, the experience is very close to
+            that of GitHub. We suggest that if you want to practice your Git skills before class that
+            you sign up for a <a href="https://github.com">GitHub account</a> and then try out
+            <a href="https://guides.github.com/activities/hello-world/">this tutorial</a>.</p>
+        </div>
+      </li>
+      <li><h4>What firewall configuration is needed?</h4>
+        <div class="answer">
+          <p>We run several services that you will need to be able to access via web browser.
+            You may need to ask your network administrator to add rules allowing access.</p>
+          <ul>
+            <li>Puppetfactory classroom manager: HTTP on port 80</li>
+            <li>Puppet Enterprise Console: HTTPS on port 443</li>
+            <li>Gitea Repository Server: HTTP on port 3000</li>
+            <li>Courseware Presentation: HTTP and Websocket on port 9090</li>
+            <li>SSH access (optional): SSH on port 22</li>
+            <li><a href="https://support.zoom.us/hc/en-us/articles/201362683-Network-Firewall-or-Proxy-Server-Settings-for-Zoom">Zoom.us network requirements</a></li>
+          </ul>
+        </div>
+      </li>
+      <li><h4>What are the browser requirements?</h4>
+        <div class="answer">
+          <p>Any modern browser should work. In general, we recommend one of the following.</p>
+          <ul>
+            <li>Relatively current versions of Firefox, Chrome, or Safari.</li>
```

Comment:
Silly question, but do we have approximate cutoffs for older versions of Firefox, Chrome, and Safari?  Or do we not want to include that because then we'll have to constantly be checking and updating this?

Reply:
I don't want to maintain an exact matrix of compatibility, because then we'd have to audit all tooling (gitea, console, puppetfactory, zoom, etc.)